docs/source/open-edx-background.rst

```diff
@@ -0,0 +1,36 @@
+Open edX Background
+===================
+
+What is Open edX?
+--------------------
+Open edX refers to its two components: the Open edX Platform and the Open edX Community.
+The Open edX Platform is a scaleable and learner centered web-based platform that can be used to author and manage courses.
```

Comment:
I might extend this sentence a bit to be very clear that this platform is both for course authors and learners - using the well known term LMS (learning management system) and the kind of well known term CMS (content management system).

Reply:
Sentence has been extended